src/DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -648,6 +657,16 @@ function AnyEntryMapFixes()
                 break;
             }
         }
+
+        // TODO: make work
```

Comment:
Explained how to go about doing this here: https://discord.com/channels/823629359931195394/823629734457114654/1382023103583883355